pyproject.toml

```diff
@@ -61,38 +61,44 @@ tests = [
     "scipy==1.10.1; python_version < '3.9'", # to support python 3.8
     "scipy==1.11.1; python_version > '3.8'",
     "pandas==2.0.3",
+    "pyiges==0.3.1",
+    "geomdl==5.3.1",
     "pytest==7.4.0",
     "pytest-cov==4.1.0",
-    "pyvista==0.39.1",
+    "pyvista==0.40.0",
     "pyansys-tools-report==0.6.0",
     "vtk==9.2.6",
     "pytest-rerunfailures==11.1.2",
 ]
 doc = [
     "sphinx==6.2.1",
     "ansys-dpf-core==0.8.1",
-    "ansys-mapdl-reader==0.52.16",
+    "ansys-mapdl-reader==0.52.17",
     "ansys-sphinx-theme==0.9.9",
     "grpcio==1.51.1",
     "imageio-ffmpeg==0.4.8",
     "imageio==2.31.1",
     "jupyter_sphinx==0.4.0",
     "jupyterlab>=3.2.8",
     "matplotlib==3.7.1",
+    "nest-asyncio==1.5.6",
     "numpydoc==1.5.0",
     "pandas==2.0.3",
     "plotly==5.15.0",
+    "pyiges==0.3.1",
+    "geomdl==5.3.1",
     "pypandoc==1.11",
     "pytest-sphinx==0.5.0",
     "pythreejs==2.4.2",
-    "pyvista==0.39.1",
+    "pyvista==0.40.0",
```

Comment:
Why not use ``pyvista[trame]``?

Reply:
I have nothing against it. Do you recommend it?

Reply:
If you want to use trame, for sure

Reply:
I'm going to do that in another PR :)